### PR TITLE
[NOJIRA] Update ESLint config

### PR DESCRIPTION
**Pull request by john-sonz**
ESLint config update. Any suggestions for other rules are welcome!

---

#### Resources
Available rules:
https://eslint.org/docs/rules/
https://github.com/jsx-eslint/eslint-plugin-react#list-of-supported-rules
https://www.npmjs.com/package/@typescript-eslint/eslint-plugin

### CODE REVIEW

File: cabra/.eslintrc.json [hunk 1]
```diff
@@ -17,7 +17,15 @@
   },
   "plugins": ["@typescript-eslint", "prettier"],
   "rules": {
+    "camelcase": "error",
+    "eqeqeq": "error",
```
Comment (john-sonz) [inline cabra/.eslintrc.json:21]: Dedicated to @tgargula ❤️

Comment (tgargula) [reply cabra/.eslintrc.json:21]: <3

**Review by tgargula — APPROVED** [review]
LGTM

**Review by koszar91 — APPROVED** [review]
Looks fine 💪

**Pull request merged**